clusters/hive/hypershift.yaml

```diff
@@ -0,0 +1,97 @@
+apiVersion: v1
```

Comment:
I imagine you'll want to merge this separately so that we can get some rehearsals in this PR?